source/extensions/access_loggers/wasm/BUILD

```diff
@@ -0,0 +1,38 @@
+load(
+    "//bazel:envoy_build_system.bzl",
+    "envoy_cc_extension",
+    "envoy_cc_library",
+    "envoy_extension_package",
+)
+
+licenses(["notice"])  # Apache 2
+
+envoy_extension_package()
+
+# Access log implementation that calls into a WASM VM.
+
+envoy_cc_library(
+    name = "wasm_access_log_lib",
+    hdrs = ["wasm_access_log_impl.h"],
+    deps = [
+        "//include/envoy/access_log:access_log_interface",
+        "//source/common/http:header_map_lib",
+    ],
+)
+
+envoy_cc_extension(
+    name = "config",
+    srcs = ["config.cc"],
+    hdrs = ["config.h"],
+    security_posture = "unknown",
```

Comment:
You probably also want to tag this as alpha.

Reply:
Done